docs/useCases.md

```diff
@@ -52,6 +52,10 @@ The different use cases currently available in the package are classified below,
 - [Users](#Users)
   - [Users read use cases](#users-read-use-cases)
     - [Get Current Authenticated User](#get-current-authenticated-user)
+    - [Get Current API Token](#get-current-api-token)
+  - [Users write use cases](#users-write-use-cases)
+    - [Delete Current API Token](#delete-current-api-token)
+    - [Recreate Current API Token](#recreate-current-api-token)
```

Comment:
I think we arre missing one use case for **creating** an API Token, or maybe should we use for that the Recreate use case also?

Reply:
Yes, the same use case is used when creating an API Token and there is not an existing one.